src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -204,8 +214,10 @@ Template:
 ```Text
 sub-<label>/[ses-<label>/]
     func/
-        sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_ce-<label>][_dir-<label>][_rec-<label>][_run-<index>][_echo-<index>]_<contrast_label>.nii[.gz]
-        sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_ce-<label>][_dir-<label>][_rec-<label>][_run-<index>][_echo-<index>]_sbref.nii[.gz]
+        sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_ce-<label>][_dir-<label>][_rec-<label>][_run-<index>][_echo-<index>]_bold.nii[.gz]
+        sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_ce-<label>][_dir-<label>][_rec-<label>][_run-<index>][_echo-<index>]_phase.nii[.gz]
+        sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_ce-<label>][_dir-<label>][_rec-<label>][_run-<index>][_echo-<index>][_part-<mag|phase>]_cbv.nii[.gz]
+        sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_ce-<label>][_dir-<label>][_rec-<label>][_run-<index>][_echo-<index>][_part-<mag|phase>]_sbref.nii[.gz]
```

Comment:
I am not sure if such detailed specification and expansion of different `contrast_label`s is needed here. 
`_bold` is indeed a special case though since now it has a dedicated `_phase`.  So now there cannot be another `func` contrast than `_bold` which would have its `_phase`. 
NB ASL BEP005 seems decided to avoid `func/` and went for a dedicted `perf/` (perfusion) although its purpose AFAIK would be `func/`.
Unfortunately I did not follow the discussion in #128.  I would have voted for any `contrast_label` to "assume" its default "composition" - either it is a mag or phase or combined if no `_part` is provided.  Then #128 could have resulted in adding `_part-phase` and having explicit `_part-phase_bold`, opening possibilities for any other `_part-phase_<contrast>`.

Sure thing `_part-` makes no sense with `_phase`, but I think that is just a corner case which might be gone in 2.0.
So why not just to keep `[_part-<mag|phase>]_<contrast_label>` and just add a note to `_phase` in the table that `_part` then should not be specified. Also it might be worth starting adding "deprecation" statements on elements which will be removed in 2.0, but that would be a separate issue I guess ;)